android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -1,6 +1,7 @@
 package com.devcycle.sdk.android.api
 
 import android.content.Context
+import android.util.Log
```

Comment:
just change from wildcard import to only Log? looks good otherwise :)

Reply:
removed the import